code/modules/projectiles/projectile/bullets.dm

```diff
@@ -53,6 +53,18 @@
 /obj/item/projectile/bullet/heavybullet
 	damage = 35
 
+/obj/item/projectile/bullet/sabot
+	damage = 150
+	var/breakthings = FALSE
+	armour_penetration = -5 //NOT A TYPO
+
+/obj/item/projectile/bullet/sabot/Range()
+	..()
+	damage += 70

```

Comment:
Isn't Range() on every turf it passes, and won't that result in someone two tiles away from you dying in one hit